cypress.config.js

```diff
@@ -0,0 +1,18 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  fixturesFolder: 'tests/cypress/fixtures',
+  screenshotsFolder: 'tests/cypress/screenshots',
+  videosFolder: 'tests/cypress/videos',
+  downloadsFolder: 'tests/cypress/downloads',
+  pageLoadTimeout: 300000,
+  e2e: {
+    setupNodeEvents(on, config) {
+      // implement node event listeners here
+    },
+    baseUrl: 'http://cdash:8080',
```

Comment:
@williamjallen, builds okay but running the tests inside the docker container required a manual change on this line to `localhost` instead of `cdash` 

```
11:   0 passing (9s)
11:   1 failing
11: 
11:   1) template spec
11:        passes:
11:      CypressError: Timed out retrying after 4000ms: The command was expected to run against origin `http://cdash:8080` but the application is at origin `http://localhost:8080`.
```

Are there plans to attempt to recognize when each value is needed?

Reply:
This is a tricky situation at the moment.

In our CI, `APP_URL` is set to `http://cdash:8080`, because the Protractor tests access the site from outside the Docker container.  Locally, you want to set `APP_URL` to `http://localhost:8080` because you want it to work on your host machine.  This configuration option must match the `APP_URL` environment variable.

Having to change your `APP_URL` every time you want to run the tests locally isn't a good solution though as you point out, so I'll see what I can do.  CMake is not currently able to parse `.env` files as far as I am aware, but I may be able to parse the `APP_URL` out of the file somehow.

Reply:
It now uses whatever you specified as the `APP_URL`.  It falls back to `http://localhost:8080` in case you are running Cypress locally outside of CTest.